node/src/accountant/mod.rs

```diff
@@ -2158,9 +2408,9 @@ mod tests {
             .begin_scan_params(&begin_scan_params_arc)
             .begin_scan_result(Err(BeginScanError::NothingToProcess))
             .begin_scan_result(Ok(QualifiedPayablesMessage {
-                protected_qualified_payables: protect_payables_in_test(vec![make_payable_account(
-                    123,
-                )]),
+                protected_qualified_payables: protect_qualified_payables_in_test(vec![
+                    make_non_guaranteed_qualified_payable(123),
```

Comment:
Let's call it `make_meaningless_qualified_payable`. Remove the `guaranteed` from the other function and call it `make_qualified_payable`.

Suggestion by @bertllll